src/__tests__/supadata.test.ts

```
@@ -1,10 +1,11 @@
-import { Supadata } from '../index.js';
 import fetchMock from 'jest-fetch-mock';
+import { Supadata } from '../index.js';
 import type {
+  Map,
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)